packages/components/src/Filter/FilterButton/subcomponents/FilterButtonBase/FilterButtonBase.module.scss

```diff
@@ -2,7 +2,7 @@
 @import "~@kaizen/design-tokens/sass/color";
 @import "~@kaizen/design-tokens/sass/typography";
 @import "~@kaizen/design-tokens/sass/spacing";
-@import "../../../../styles/button-reset";
+@import "../../../../../styles/button-reset";
```

Comment:
OOS: Might be nice if we made some aliases for these lol

Reply:
I think the only way we can alias these is if we made it its own package, cause I don't think SCSS has an alias function 🤔

Reply:
Pretty sure it does, but where the config lives is the question 😬 For another day